packages/edit-site/src/components/editor/style.scss

```diff
@@ -0,0 +1,23 @@
+.edit-site-editor__toggle-save-panel {
+	z-index: z-index(".edit-site-editor__toggle-save-panel");
+	position: fixed !important; // Need to override the default relative positionning
```

Comment:
```suggestion
	position: fixed !important; // Need to override the default relative positioning
```

Reply:
Ah yeah, thats what I get for copying styles over from the other a11y buttons and only vaguely reading the comments.